src/reset.css

```diff
@@ -0,0 +1,137 @@
+/* stylelint-disable selector-pseudo-element-colon-notation */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  /* stylelint-disable-next-line declaration-block-no-shorthand-property-overrides */
+  font: inherit;
+  vertical-align: baseline;
+}
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+body {
+  line-height: 1;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+ol,
+ul {
+  list-style: none;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+blockquote,
+q {
+  quotes: none;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+a,
+ul {
+  text-decoration: none;
+  color: black;
```

Comment:
Setting a universal color of black for `a` and `ul` might not be suitable for all use cases. Ensure that this does not conflict with your design requirements elsewhere in your stylesheets.